src/Microsoft.Data.Analysis/DataFrame.cs

```diff
@@ -32,20 +32,28 @@ public enum DropNullOptions
     public partial class DataFrame
     {
         private readonly DataFrameColumnCollection _columnCollection;
+        private readonly DataFrameRowCollection _rowCollection;
         public DataFrame()
         {
             _columnCollection = new DataFrameColumnCollection(OnColumnsChanged);
+            _rowCollection = new DataFrameRowCollection(this);
         }
 
         public DataFrame(IList<DataFrameColumn> columns)
         {
             _columnCollection = new DataFrameColumnCollection(columns, OnColumnsChanged);
+            _rowCollection = new DataFrameRowCollection(this);
         }
 
         public long RowCount => _columnCollection.RowCount;
 
         public DataFrameColumnCollection Columns => _columnCollection;
 
+        /// <summary>
+        /// Returns a <see cref="DataFrameRowCollection"/> that contains the rows in this <see cref="DataFrame"/>
```

Comment:
```suggestion
        /// Returns a <see cref="DataFrameRowCollection"/> that contains a view to the rows in this <see cref="DataFrame"/>
```
?

BTW: for other members the xml-comments are missing.